core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1071,7 +1071,7 @@ private void processWorkerAddedBlocks(MasterWorkerInfo workerInfo,
             mBlockStore.addLocation(blockId, location);
             mLostBlocks.remove(blockId);
           } else {
-            LOG.warn("Invalid block: {} from worker {}.", blockId,
+            LOG.debug("Invalid block: {} from worker {}.", blockId,
```

Comment:
have one info log indicating the total number of invalid block for this worker?

Reply:
Good call! Does WARN level make more sense for both?

Reply:
> Good call! Does WARN level make more sense for both?

That shall work.